### PR TITLE
add 2.9.1 release notes

### PR DESCRIPTION
**Pull request by Mojachieee**

### CODE REVIEW

**Review by geordieintheshellcode — APPROVED** [review]

**Review by hubvu — APPROVED** [review]
LGTM

File: docs/release-notes.md [hunk 1]
```diff
@@ -9,14 +9,40 @@ description: >
...
+* CSI helper timeouts can now be configured via the StorageosCluster CRD
+* CSI Provisioner’s number of worker threads can now be configured via the StorageosCluster CRD
+* CSI Provisioner’s number of worker threads now defaults to 32 (was 100, which caused issues when creating many volumes at once)
+* CSI Provisioner’s timeout now defaults to 30s (was 10s, which occasionally caused issues when creating many volumes at once)
+* Dataplane no longer unnecessarily serialises concurrent device creation. This helps when creating multiple volumes in parallel under certain scenarios.
+
+### Fixed
+
+* Significantly reduce the CPU usage when creating volumes
+  * This fixes an issue where creating many volumes at once could cause small clusters to become unstable
```
Comment (chira001) [inline docs/release-notes.md:37]: I'd suggest removing the sub-bullet

File: docs/release-notes.md [hunk 2]
```diff
@@ -9,14 +9,40 @@ description: >
...
 [DockerHub](https://hub.docker.com/r/storageos/cli).
 
+## 2.9.1 - Release 2022-11-23
+
+This release improves stability and reduces resource consumption when creating many volumes at once.
+
+### New
+
+* Allow dataplane startup timeout to be configured
+* Increase control plane's default max log size to 25MiB and enable log compression
```
Comment (Arau) [inline docs/release-notes.md:27]: 25MiB? or 256?

Comment (Mojachieee) [reply docs/release-notes.md:27]: 25 MiB, but there can be up to 10 log files, so the max total size is 250 MiB (though it'll likely be much smaller due to compression). 

I'll update this note to clarify

**Pull request merged**